src/www/test.action.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -eu
```

Comment:
I move the test script into the www folder as those were the only things we were testing. 

I was thinking - should `www` be called `browser`, here? To minimize confusing it with the root `www` ?

Reply:
What do you think moving all test specs and code into `./test/` folder? For example, `test/www`

Reply:
Not opposed, but I'm a fan of co-locating stuff by concern so it's more fungible. That way I can move `www` somewhere else and don't have to remember to update my `test` folder (along with any others) to point to it. This is also more of a npm workspace/monorepo pattern, where each package carries its own tests. That way we can incrementally run them based on which packages change.

Reply:
Moving to src/www makes sense.

I'd call webpack in this file though. We need a different build. For instance, no need for the environment json.

I'm not a big fan of having a separate tree for tests a the root. Within a component that's fine. One of the reasons of using Webpack is that it knows what to pull, so we don't need to worry if files are mixed together.

Reply:
Yeah, that will solve the issue of needing the erroneous `test` platform. Thanks.